### PR TITLE
docs: update codelabs

### PR DESCRIPTION
**Pull request by yzdocs**
Release note: None

Correct outdated information in SQL statement code lab.

### CODE REVIEW

File: docs/codelabs/01-sql-statement.md [hunk 1]
```diff
@@ -48,7 +48,7 @@ CLUSTER`, which operates on cluster settings, `FROBNICATE SESSION`, working on
 session settings, and `FROBNICATE ALL`, which handles both.
 
 Let's start by checking to make sure all our keywords are defined.  Open
-`pkg/sql/parser/sql.y` and search for "keyword".  You'll find a series of token
+`pkg/sql/parser/sql.y` and search for "Ordinary key words".  You'll find a series of token
```
Comment (rohany) [inline docs/codelabs/01-sql-statement.md:51]: The search should be for `unreserved_keyword`

Comment (yingzhuchin) [reply docs/codelabs/01-sql-statement.md:51]: This is for adding the new keyword to the section starting line 554 in `sql.y`. Further down in this code lab (line 65) you'll see that we guide people to add specifically to the `unreserved_keyword` list.

Comment (rohany) [reply docs/codelabs/01-sql-statement.md:51]: Oh i see. Looks good then.

**Review by rohany — APPROVED** [review]

File: docs/codelabs/01-sql-statement.md (hunk 1 above)
Comment (yingzhuchin) [reply docs/codelabs/01-sql-statement.md:51]: Thanks for reviewing!

**Comment by yingzhuchin** [comment]
Quick comment that I'm closing this PR in favor of https://github.com/cockroachdb/cockroach/pull/52012, because I forgot to push from my fork instead of directly (-__- sorry). Reviewer approvals carry over, so no need to re-approve there.

**Pull request closed without merging**